### PR TITLE
Update_CLOAD_and_PLOAD

### PR DESCRIPTION
**Pull request by mircea-altair**
#### Description of the feature or the bug

Update /CLOAD and /PLOAD to allow Force vs displacement/velocity loads.

#### Description of the changes

Update /CLOAD and /PLOAD to allow Force vs displacement/velocity loads.

### CODE REVIEW

File: engine/source/output/h3d/h3d_results/h3d_skin_scalar.F [hunk 1]
```diff
@@ -81,7 +81,7 @@ SUBROUTINE H3D_SKIN_SCALAR(
       INTEGER  ,INTENT(IN) :: NSENSOR
       my_real
      .   SKIN_SCALAR(*),X(3,*),V(3,*),W(3,*),GEO(NPROPG,*),PM(NPROPM,*),
-     .   TF(*),BUFMAT(*)
+     .   TF(*),BUFMAT(*),D(3,NUMNOD)
```
Comment (sebastienVilleneuve) [inline engine/source/output/h3d/h3d_results/h3d_skin_scalar.F:84]: INTENT statement is missing

File: engine/source/output/h3d/h3d_results/h3d_skin_scalar.F [hunk 2]
```diff
@@ -547,7 +547,7 @@ SUBROUTINE H3D_PRE_SKIN_SCALAR(SKIN_SCALAR,NODAL_IPART,
 C-----------------------------------------------
       INTEGER  ,INTENT(IN) :: NSENSOR
       my_real
-     .   SKIN_SCALAR(*),TF(*),X(3,*),V(3,*)
+     .   SKIN_SCALAR(*),TF(*),X(3,*),V(3,*),DIS(3,NUMNOD)
```
Comment (sebastienVilleneuve) [inline engine/source/output/h3d/h3d_results/h3d_skin_scalar.F:550]: INTENT statement is missing

File: engine/source/output/h3d/h3d_results/h3d_skin_scalar.F [hunk 3]
```diff
@@ -628,17 +629,44 @@ SUBROUTINE H3D_PRE_SKIN_SCALAR(SKIN_SCALAR,NODAL_IPART,
...
+             X_OLD = TS
+           ENDIF
+         ELSE IF(FUNCTYPE == 2) THEN
+           IF(N_OLD/=N5) THEN
+             DISP = (DIS(3,N1)+DIS(3,N2)+DIS(3,N3)+DIS(3,N4))/4.0
+             ISMOOTH = 0
+             IF (N5 > 0) ISMOOTH = NPC(2*NFUNCT+N5+1)
+             IF (ISMOOTH == 0) THEN
+               F1 = FINTER(N5,DISP*FCX,NPC,TF,DYDX)
+              ELSE
```
Comment (sebastienVilleneuve) [inline engine/source/output/h3d/h3d_results/h3d_skin_scalar.F:651]: seems indentation is wrong

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
hi @mircea-altair 

some minor changes to perform.

Thanks,
Sébastien

File: engine/source/output/h3d/h3d_results/h3d_skin_scalar.F (hunk 3 above)
Comment (mircea-altair) [reply engine/source/output/h3d/h3d_results/h3d_skin_scalar.F]: corrected

**Review by sebastienVilleneuve — APPROVED** [review]

**Pull request merged**